src/main/frontend/src/components/entrypoints/versions/CreateVersion.vue

```diff
@@ -370,7 +370,7 @@ export default {
             alert.find('.file-size').text(filesize(files[0].size));
             alert.fadeIn('slow');
 
-            $('#form-url-upload').css('display', 'none');
+            document.querySelector('form-url-upload').style.display = "none";
```

Comment:
I think this needs to have a `#` before it right?

Reply:
No it needs to be a getElementById. Thanks though